Elements/src/Geometry/Line.cs

```diff
@@ -850,6 +880,49 @@ public Line ExtendTo(Polygon polygon, double maxDistance, bool bothSides = true,
             return ExtendTo(polygon.Segments(), maxDistance, bothSides, extendToFurthest, tolerance);
         }
 
+        /// <summary>
+        /// Measures the distance between two lines.
+        /// Computes a cross product of the lines' directional vectors.
+        /// Case 1: It's zero, meaning the lines are collinear, which means we can simply orthogonalize a vector that connects points on different line wrt the directional vector.
+        /// Case 2: It's not zero, meaning the moving of the points on the two lines spans a 2-dimensional subspace of a 3-dimensional space, so the shortest segment connecting the lines will be collinear to the cross product.
+        /// </summary>
+        /// <param name="other">The line to measure the distance to</param>
+        /// <returns></returns>
+        public double DistanceTo(Line other)
+        {
+            Vector3 dist = this.Start - other.Start;
+            Vector3 v1 = this.End - this.Start;
```

Comment:
Lets try to be little bit more specific with variable name in the future. I think next names, for example, will add to readability:
v1 => thisDelta
v2 => cross
v3 => otherStart;
v4 => otherEnd;
...
v3 => startToStart
v4 => startToEnd 
and so on.
The name are not final just wanted to give you an idea of balance between length and meaning